src/commands/seek.ts

```diff
@@ -213,7 +213,9 @@ export function word(
   const charset = ws ? CharSet.NonBlank : CharSet.Word;
 
   Selections.updateWithFallbackByIndex((_i, selection) => {
-    const anchor = Selections.seekFrom(selection, direction, selection.anchor, _);
+    const wasreversed = selection.isReversed;
```

Comment:
nit: please use `camelCase`

(Out of curiosity, what language do you usually code in? I don't think I've ever seen this case outside of Python).

Reply:
You guessed correctly, mostly Python

Reply:
shall I fix the casing before you merge?

Reply:
Yes, please do!

Reply:
Done